### ISSUE TITLE
Install fails without VictorPlexus login despite manually downloading all commonly failed downloads as per instruction

### ISSUE THREAD

**Issue by Wesleypipes77** [zhongjiedong/Tsukiro#118]
### What happened?

Just like the title says, I already downloaded and moved all "Commonly failed downloads" to the appropriate folder. But when trying to install Wabbajack still wants me to login to Victorplexus and fails when I don't/can't.

### Modlist version

I don't know, whichever one wabbajack decides to download I guess. I would imagine the latest 1.1.74?

**Comment by catchra** [comment]
I have the same problem, Vector Plexus web site down there is a mirror site but wabbajack can't use it

**Comment by catchra** [comment]
lol never mine it will still fail

```2837.67 - Getting Nexus API Key, if a browser appears, please accept
2837.67 - Loading server whitelist
2837.92 - Loaded permissions for 709 servers and 312 Google Drive files
2837.92 - Getting upgrades list
2838.07 - File hash check (-42) c9819e9e504aad97a1c02eca2b337166ba173085f5f07a10e13aa717a4c35f33
2838.11 - Downloading Disable Turn Animation.7z
2838.11 - Downloading tsukirotexgen043.7z
2838.11 - Downloading GS_SLAL SE 04 07.7z
2838.11 - Downloading tsukirimlodgen043.7z
2838.11 - Downloading Smooth Combat Non Combat Animation.7z
2838.11 - Downloading New cloud YOU NEED THAT.rar
2838.11 - Downloading Smooth Magic Casting Animation 3.2.7z
2838.11 - Downloading Yamete! V3.2.7z
2838.18 - Loading for alternative to 5/0OC/4vYR0=
2838.18 - Loading for alternative to v3ysDsnP6Rs=
2838.18 - Loading for alternative to gOjM28PxoJM=
2838.18 - No alternative for 5/0OC/4vYR0= could be found
2838.18 - No alternative for v3ysDsnP6Rs= could be found
2838.18 - No alternative for gOjM28PxoJM= could be found
2838.19 - Downloading tsukirograss060.7z
2838.19 - Downloading Nemesis.Unlimited.Behavior.Engine.v0.84-beta.rar
2838.19 - Downloading immyneedscake - RyanReos Elf Paladin-44405-1-2-1610945561.7z
2838.27 - Loading for alternative to j0QG36DeIp4=
2838.27 - No alternative for j0QG36DeIp4= could be found
2838.27 - Downloading Yamete! V3.1.7z
2838.32 - Loading for alternative to PjjK6QqKKE0=
2838.32 - No alternative for PjjK6QqKKE0= could be found
2838.32 - Downloading tsukirodyndolod043.7z
2838.32 - Loading for alternative to XzyZeCQLvYA=
2838.32 - No alternative for XzyZeCQLvYA= could be found
2838.32 - Downloading SexLab Utility Plus v20210315.7z
2838.34 - Loading for alternative to 5gZ8pQg4qk8=
2838.34 - No alternative for 5gZ8pQg4qk8= could be found
2838.34 - Downloading NCK30 SLAL 3.0 SE.7z
2838.4 - File hash check (-42) c9819e9e504aad97a1c02eca2b337166ba173085f5f07a10e13aa717a4c35f33
2838.48 - Loading for alternative to KwNmGoBj31s=
2838.48 - No alternative for KwNmGoBj31s= could be found
2838.48 - Downloading LibMathf-49896-1-0-0-1620959710.zip
2838.55 - Loading for alternative to uq8KT2b+f9A=
2838.55 - No alternative for uq8KT2b+f9A= could be found
2838.55 - Downloading LibFire-49897-1-0-7-1-1623190394.zip
2838.6 - Loading for alternative to dr3CFmJ9K+o=
2838.6 - No alternative for dr3CFmJ9K+o= could be found
2838.6 - Downloading Courier Shows Delivery USSEP Fix-51447-1-0-1-1624215817.zip
2838.7 - File hash check (-42) c9819e9e504aad97a1c02eca2b337166ba173085f5f07a10e13aa717a4c35f33
2838.72 - Loading for alternative to ggf8bGm3k9w=
2838.72 - No alternative for ggf8bGm3k9w= could be found
2838.72 - Downloading Courier Shows Delivery-51447-1-0-1-1624215656.zip
2838.74 - Loading for alternative to aIGLhIhgwdw=
2838.75 - No alternative for aIGLhIhgwdw= could be found
2838.75 - Downloading bodyslideoutput7.7z
2838.77 - Loading for alternative to N9wnhZbFk+Y=
2838.77 - No alternative for N9wnhZbFk+Y= could be found
2838.77 - Downloading bodyslideoutput6.7z
2838.82 - Loading for alternative to uQJKYy7hSP4=
2838.82 - No alternative for uQJKYy7hSP4= could be found
2838.82 - Downloading bodyslideoutput5.7z
2838.82 - Loading for alternative to ewKZmzOMG9o=
2838.82 - No alternative for ewKZmzOMG9o= could be found
2838.82 - Downloading bodyslideoutput4.7z
2838.82 - Loading for alternative to rgT4GkjzwSA=
2838.82 - No alternative for rgT4GkjzwSA= could be found
2838.83 - Downloading bodyslideoutput3.7z
2838.83 - Loading for alternative to APwRHMsGBso=
2838.83 - No alternative for APwRHMsGBso= could be found
2838.83 - Downloading bodyslideoutput2.7z
2838.84 - Loading for alternative to ghvnOSYb3MQ=
2838.84 - No alternative for ghvnOSYb3MQ= could be found
2838.84 - Downloading bodyslideoutput1.7z
2838.84 - Loading for alternative to VrWaZloEJRg=
2838.84 - No alternative for VrWaZloEJRg= could be found
2838.84 - Downloading bodyslideoutput0.7z
2838.85 - File hash check (-42) c9819e9e504aad97a1c02eca2b337166ba173085f5f07a10e13aa717a4c35f33
2838.85 - Loading for alternative to pFNJfeuGa5M=
2838.85 - No alternative for pFNJfeuGa5M= could be found
2838.85 - Downloading 944402306_SLAL_Billyy_Animationsv4.6SE.zip
2838.87 - Loading for alternative to 0aQabGFeO2A=
2838.88 - Loading for alternative to yseZLqnHC5E=
2838.88 - Loading for alternative to dhg8oDm/9D4=
2838.88 - No alternative for 0aQabGFeO2A= could be found
2838.88 - No alternative for yseZLqnHC5E= could be found
2838.88 - No alternative for dhg8oDm/9D4= could be found
2838.88 - Downloading SOS - ERF - High Poly Addon BHUNP.7z
2838.88 - Downloading 766663625_BakaFactoryABCLESE2.26V.7z
2838.9 - Loading for alternative to tXqDCQrY4uA=
2838.9 - No alternative for tXqDCQrY4uA= could be found
2838.9 - Loading for alternative to zqIE6GhcSiE=
2838.9 - No alternative for zqIE6GhcSiE= could be found
2838.92 - Loading for alternative to 0F1luNS9eCI=
2838.92 - No alternative for 0F1luNS9eCI= could be found
2838.93 - Loading for alternative to StyuQ/ud9eE=
2838.93 - No alternative for StyuQ/ud9eE= could be found
2839.1 - Loading for alternative to EahjtJ9Jsng=
2839.1 - No alternative for EahjtJ9Jsng= could be found
2839.11 - Loading for alternative to WT3l0uuUGqI=
2839.11 - No alternative for WT3l0uuUGqI= could be found
2839.2 - Loading for alternative to UkJ1Hx9/HyQ=
2839.2 - No alternative for UkJ1Hx9/HyQ= could be found
2839.24 - Hashing Remaining Archives
2839.24 - Looking for files to hash
2839.34 - Found 815 total files, 643 matching filesize
2839.58 - Unable to download 766663625_BakaFactoryABCLESE2.26V.7z (LoversLabOAuthDownloader+State|7556|766663625_BakaFactoryABCLESE2.26V.7z|False)
2839.59 - Unable to download SOS - ERF - High Poly Addon BHUNP.7z (HTTPDownloader+State|https://www.patreon.com/file?h=37684959&i=5705399)
2839.59 - Unable to download 944402306_SLAL_Billyy_Animationsv4.6SE.zip (LoversLabOAuthDownloader+State|3999|944402306_SLAL_Billyy_Animationsv4.6SE.zip|False)
2839.59 - Unable to download bodyslideoutput0.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput0.7z_7878a94e-72b5-47e0-b104-a07c9f218b56)
2839.59 - Unable to download bodyslideoutput1.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput1.7z_a576f79a-f784-431e-8263-42bab9826f44)
2839.59 - Unable to download bodyslideoutput2.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput2.7z_583698b5-b1cf-4f34-a27e-7d1502c589c9)
2839.59 - Unable to download bodyslideoutput3.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput3.7z_bb228086-c03a-443e-9487-c8df047dcbd8)
2839.59 - Unable to download bodyslideoutput4.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput4.7z_0941b0ce-9857-4604-8558-f3c2f975a20b)
2839.59 - Unable to download bodyslideoutput5.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput5.7z_eaa71e1c-1c98-4929-81ff-7b17b28c5be9)
2839.59 - Unable to download bodyslideoutput6.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput6.7z_b49b6354-09a6-47a0-a9b6-4d5676abed60)
2839.59 - Unable to download bodyslideoutput7.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/bodyslideoutput7.7z_106ada15-86d8-4c8d-a71f-c780b08135b6)
2839.59 - Unable to download Courier Shows Delivery-51447-1-0-1-1624215656.zip (NexusDownloader+State|SkyrimSpecialEdition|51447|210289)
2839.59 - Unable to download Courier Shows Delivery USSEP Fix-51447-1-0-1-1624215817.zip (NexusDownloader+State|SkyrimSpecialEdition|51447|210291)
2839.59 - Unable to download LibFire-49897-1-0-7-1-1623190394.zip (NexusDownloader+State|SkyrimSpecialEdition|49897|208200)
2839.59 - Unable to download LibMathf-49896-1-0-0-1620959710.zip (NexusDownloader+State|SkyrimSpecialEdition|49896|203359)
2839.59 - Unable to download NCK30 SLAL 3.0 SE.7z (LoversLabOAuthDownloader+State|6878|NCK30 SLAL 3.0 SE.7z|False)
2839.59 - Unable to download SexLab Utility Plus v20210315.7z (LoversLabOAuthDownloader+State|10316|SexLab Utility Plus v20210315.7z|False)
2839.59 - Unable to download tsukirodyndolod043.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/tsukirodyndolod043.7z_eccaa385-1db2-46b2-b22c-7b7d1c1697fb)
2839.6 - Unable to download Yamete! V3.1.7z (LoversLabOAuthDownloader+State|16057|Yamete! V3.1.7z|False)
2839.6 - Unable to download immyneedscake - RyanReos Elf Paladin-44405-1-2-1610945561.7z (NexusDownloader+State|SkyrimSpecialEdition|44405|180188)
2839.6 - Unable to download Nemesis.Unlimited.Behavior.Engine.v0.84-beta.rar (HTTPDownloader+State|https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/releases/download/v0.84-beta/Nemesis.Unlimited.Behavior.Engine.v0.84-beta.rar)
2839.6 - Unable to download tsukirograss060.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/tsukirograss060.7z_723fdc7b-a5a8-44b0-9d58-f3e7b1ffc039)
2839.6 - Unable to download Yamete! V3.2.7z (LoversLabOAuthDownloader+State|16057|Yamete! V3.2.7z|False)
2839.6 - Unable to download Disable Turn Animation.7z (HTTPDownloader+State|https://blog.kakaocdn.net/dn/H1jDr/btq4oxONgle/6ENV1nGqob6zw5XHbevwqk/Disable%20Turn%20Animation.7z?attach=1&knm=tfile.7z)
2839.6 - Unable to download New cloud YOU NEED THAT.rar (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/New cloud YOU NEED THAT.rar_726ea70c-89b4-4d46-87c4-69e63d897299)
2839.6 - Unable to download tsukirotexgen043.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/tsukirotexgen043.7z_6ca270d9-a70c-4247-9ada-7acd12777923)
2839.6 - Unable to download GS_SLAL SE 04 07.7z (LoversLabOAuthDownloader+State|8148|GS_SLAL SE 04 07.7z|False)
2839.6 - Unable to download tsukirimlodgen043.7z (WabbajackCDNDownloader+State|https://authored-files.wabbajack.org/tsukirimlodgen043.7z_de4f6f04-cc1c-4268-a13f-32f4b8fddb4c)
2839.6 - Unable to download Smooth Magic Casting Animation 3.2.7z (HTTPDownloader+State|https://blog.kakaocdn.net/dn/cyDuii/btq74ntfh9i/FwXmgAkFKizXGmtyVpscZK/Smooth%20Magic%20Casting%20Animation%203.2.7z?attach=1&knm=tfile.7z)
2839.6 - Unable to download Smooth Combat Non Combat Animation.7z (HTTPDownloader+State|https://blog.kakaocdn.net/dn/9Tn0e/btq6pI8JGgq/RMxvJoTOF071qWFu9pc04k/Smooth%20Combat%20Non%20Combat%20Animation.7z?attach=1&knm=tfile.7z)
2839.6 - Cannot continue, was unable to download one or more archives
2839.6 - File hash check (-42) c9819e9e504aad97a1c02eca2b337166ba173085f5f07a10e13aa717a4c35f33
2839.6 - Encountered error, can't continue - System.Exception: Cannot continue, was unable to download one or more archives
   at Wabbajack.Lib.AInstaller.Error(String msg) in c:\Wabbajack\splitwork\wabbajack\Wabbajack.Lib\AInstaller.cs:line 71
   at Wabbajack.Lib.MO2Installer._Begin(CancellationToken cancel) in c:\Wabbajack\splitwork\wabbajack\Wabbajack.Lib\MO2Installer.cs:line 161
   at Wabbajack.Lib.ABatchProcessor.<Begin>b__66_0() in c:\Wabbajack\splitwork\wabbajack\Wabbajack.Lib\ABatchProcessor.cs:line 177
   at Wabbajack.MO2InstallerVM.<>c__DisplayClass31_0.<<Install>b__0>d.MoveNext() in c:\Wabbajack\splitwork\wabbajack\Wabbajack\View Models\Installers\MO2InstallerVM.cs:line 181
--- End of stack trace from previous location ---
   at Wabbajack.MO2InstallerVM.Install() in c:\Wabbajack\splitwork\wabbajack\Wabbajack\View Models\Installers\MO2InstallerVM.cs:line 175
   at Wabbajack.InstallerVM.<>c__DisplayClass88_0.<<-ctor>b__78>d.MoveNext() in c:\Wabbajack\splitwork\wabbajack\Wabbajack\View Models\Installers\InstallerVM.cs:line 391
2839.74 - File hash check (-42) c9819e9e504aad97a1c02eca2b337166ba173085f5f07a10e13aa717a4c35f33```
